tests/coalaHelperTest.py

```diff
@@ -0,0 +1,58 @@
+import json
+import os
+import shutil
+import unittest
+
+from coalahtml.helper import (get_file, parse_file_dict, build_file_graph,
+                              create_dir, copy_files, get_args)
+
+
+class coalaHelperTest(unittest.TestCase):
+
+    def setUp(self):
+        self.dir_name = "someRandomName"
+        self.test_dir_path = os.path.abspath(self.dir_name)
+        self.file_name = self.dir_name[::2]
+        test_section = 'default'
+        test_file_name = 'tempTest.py'
+        test_file_content = 'import this\n'
+        self.test_file_dict = {test_section: {
+            test_file_name: (test_file_content)}}
+        self.test_file_data = {test_file_name: test_file_content}
+
+    def tearDown(self):
+        pass
+
+    def test_get_file(self):
+        test_file_path = os.path.abspath(os.path.join(self.test_dir_path,
+                                                      self.file_name))
+        return_file_path = get_file(self.file_name, self.test_dir_path)
+        self.assertEqual(test_file_path, return_file_path)
+
+    def test_parse_file_dict(self):
+
+        result_file_data = parse_file_dict(self.test_file_dict)
+        self.assertEqual(result_file_data, self.test_file_data)
+
+    def test_create_dir(self):
+        self.assertEqual(create_dir(self.test_dir_path),
+                         self.test_dir_path)
+        os.rmdir(self.test_dir_path)
+
+    def test_copy_files(self):
+        s_dir = create_dir(os.path.abspath('S99'))
+        d_dir = create_dir(os.path.abspath('D99'))
```

Comment:
With contextmanager you won't have to delete it manually, so it'll be easier to go through or modify.